clients/mobile/app.config.ts

```diff
@@ -13,6 +13,11 @@ export default ({ config }: { config: ExpoConfig }): ExpoConfig => {
     APP_NAME?: string;
     APP_VERSION?: string;
     LOCATION_SERVICE_EMAIL?: string;
+    GOOGLE_MAP_API_KEY?: string;
```

Comment:
are these truly optional

Reply:
you've made everything optional. that means the app can function without these?

Reply:
No without Env app will not be functional. I fixed with this [commit](https://github.com/CraftsmenLtd/BloodConnect/pull/239/commits/ff602e89d8999788a6f0ecbd9fb785dc05851483)